ansible_ai_connect_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -374,12 +377,17 @@ export const AnsibleChatbot: React.FunctionComponent = () => {
                   ) : (
                     <></>
                   )}
+                  {isStreamingSupported() && (
+                    <div key={`scroll_div_9999`} ref={messagesEndRef} />
+                  )}
```

Comment:
When the streaming mode is used, UI will automatically scrolls to the bottom of the last chatbot message as it is being updated.  When non-streaming mode is used, UI will show at the top of the last chatbot message.

Reply:
It can be any number.  I just picked up a number, which should be bigger than the number of messages on the UI.  On lines 344-370, the code renders messages from the list and `index` is used to point to each message.  I assumed the total # of messages should be less than `9999` and used `9999` to indicate the end of messages.